### PR TITLE
Branch1

### PR DESCRIPTION
**Pull request by iluxa-com**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -24,12 +24,15 @@
...
+      <change type="MODIFICATION" beforePath="$PROJECT_DIR$/test.html" afterPath="$PROJECT_DIR$/test.html" />
       <change type="MODIFICATION" beforePath="$PROJECT_DIR$/.idea/workspace.xml" afterPath="$PROJECT_DIR$/.idea/workspace.xml" />
     </list>
     <ignored path="demoProject.iws" />
     <ignored path=".idea/workspace.xml" />
+    <ignored path="$PROJECT_DIR$/out/" />
+    <ignored path=".idea/dataSources.local.xml" />
+    <option name="EXCLUDED_CONVERTED_TO_IGNORED" value="true" />
     <option name="TRACKING_ENABLED" value="true" />
     <option name="SHOW_DIALOG" value="false" />
```
Comment (iluxa-com) [inline .idea/workspace.xml:37]: PR inline comment demo

File: .idea/workspace.xml [hunk 2]
```diff
@@ -40,10 +43,8 @@
   <component name="CreatePatchCommitExecutor">
     <option name="PATCH_PATH" value="" />
```
Comment (iluxa-com) [inline .idea/workspace.xml:44]: new one

**Comment by iluxa-com** [comment]
try again

**Comment by iluxa-com** [comment]
from web

**Comment by iluxa-com** [comment]
web

**Comment by iluxa-com** [comment]
0

**Comment by iluxa-com** [comment]
1

**Comment by maxim5** [comment]
<component name="ChangesViewManager" flattened_view="false" show_ignored="false" />

**Comment by maxim5** [comment]
<component name="ChangesViewManager" flattened_view="false" show_ignored="false" />

**Comment by maxim5** [comment]
ChangesViewManager

**Comment by maxim5** [comment]
Reply

**Comment by maxim5** [comment]
CreatePatchCommitExecutor

**Comment by maxim5** [comment]
CreatePatchCommitExecutor2

File: branch1.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+created
++1
++2web
+created
++1
+123
```
Comment (maxim5) [inline branch1.txt:6]: 123

Comment (maxim5) [inline branch1.txt]: reply

Comment (maxim5) [inline branch1.txt]: reply2

**Comment by iluxa-com** [comment]
123--

**Comment by iluxa-com** [comment]
image

File: .idea/workspace.xml (hunk 2 above)
Comment (iluxa-com) [inline .idea/workspace.xml:44]: reply in upsource inline

File: .idea/workspace.xml (hunk 1 above)
Comment (iluxa-com) [inline .idea/workspace.xml:37]: reply in timeline

**Comment by iluxa-com** [comment]
@{7b22695e-a885-478a-8e57-3034ad41fc97,admin} file comment

File: .idea/workspace.xml [hunk 4]
```diff
@@ -24,12 +24,15 @@
   </component>
   <component name="ChangeListManager">
     <list id="336dc758-5505-4f73-89ff-77ebbc4b24cc" name="Default" comment="XXX" />
-    <list default="true" id="ab9aec04-4f4f-440f-8da8-0162c207bc04" name="Ignore" comment="jdk 1.7">
-      <change type="MODIFICATION" beforePath="$PROJECT_DIR$/.idea/copyright/profiles_settings.xml" afterPath="$PROJECT_DIR$/.idea/copyright/profiles_settings.xml" />
+    <list default="true" id="ab9aec04-4f4f-440f-8da8-0162c207bc04" name="Ignore" comment="ccc&#10;">
+      <change type="MODIFICATION" beforePath="$PROJECT_DIR$/test.html" afterPath="$PROJECT_DIR$/test.html" />
```
Comment (iluxa-com) [inline .idea/workspace.xml:28]: workspace 28 line GH comment in Files changed

**Comment by iluxa-com** [comment]
asdf

**Comment by iluxa-com** [comment]
111

**Comment by iluxa-com** [comment]
333

**Comment by iluxa-c0m** [comment]
0000

File: .idea/workspace.xml (hunk 4 above)
Comment (iluxa-c0m) [reply .idea/workspace.xml:28]: WTF

**Comment by maxim5** [comment]
Good to see it finally being fixed :)

**Pull request closed without merging**